book/src/configuration.md

```diff
@@ -86,6 +85,16 @@ a simple boolean value, or a specific mapping of pairs of single characters.
 | `true` | Use the default pairs: <code>(){}[]''""``</code>
 | Mapping of pairs | e.g. `{ "(" =  ")", "{" = "}", ... }`
 
+
+### `[editor.search]` Section
+
+Search specific options.
+
+| Key | Description | Default |
+|--|--|---------|
+| `smart-case` | Enable smart case regex searching (case insensitive unless pattern contains upper case characters) | `true` |
+| `wrap-around`| Whether the search should wrap after depleting the matches | `true` |
+
```

Comment:
This seems to be sandwiched into the middle of the auto-pairs config, could you move it to after the auto-pairs config example ?

Reply:
whoops, how did that happen...
anyway, fixed it